src/_pytest/config/findpaths.py

```diff
@@ -18,7 +21,13 @@ def exists(path, ignore=EnvironmentError):
         return False
 
 
-def getcfg(args, config=None):
+def getcfg(
+    args: List[py.path.local], config: Optional[Config] = None
+) -> Tuple[
+    Optional[py.path.local],
+    Optional[py.path.local],
+    Union[py.iniconfig.SectionWrapper, Dict, None],
```

Comment:
Is it possible, instead of returning an empty dict in the fallback case, to return an empty `SectionWrapper`? This way the `Dict` in the union could be removed. (Same in `determine_setup`).

If not, can the `Dict` be narrowed? E.g. `Dict[str, str]`? (I didn't check).

Reply:
The `Dict` could be restricted to "an empty dict" IIRC.
Maybe returning an empty SectionWrapper would be sensible, and might only have been skipped to avoid some import?!

Reply:
SectionWrapper is not exported, so this looks like a error

Reply:
`py.iniconfig.SectionWrapper` is not importable...

Reply:
Backing that part out into https://github.com/pytest-dev/pytest/pull/6450.

Reply:
@RonnyPfannschmidt 
> SectionWrapper is not exported, so this looks like a error

Yes - it's what's being returned/used there though (IIRC).

Any idea about why `{}` is used?

Reply:
i suspect i used `{}` since it is close enough to a section wrapper to work, and decidedly not one since it cant return data, figuring it exactly would need history digging